test/java/com/rapleaf/jack/test_project/database_1/impl/Database1Impl.java

```diff
@@ -24,15 +24,17 @@
   
   private final BaseDatabaseConnection conn;
   private final IDatabases databases;
+  private final com.rapleaf.jack.tracking.PostQueryAction postQueryAction;
```

Comment:
Not really. I left it fully qualified in the generated code in general since in models I didn't want to accidentally collide with a reasonable table field name. Do you think it makes sense to change throughout to an `import com.rapleaf.jack.tracking.PostQueryAction` or just in contexts like this?